minishift/minishift
# Issue #2872 Remove entries from kubeconfig file on cluster deletion

Fixes #2872 


Steps to test the pull request

  1. `minishift start`
  2. `minishift delete`
  3. Check `~/.kube/config` before and after deletion to verify that the entries corresponding to the current cluster are removed.
